Eventy/src/main/java/org/example/eventy/solutions/controllers/ProductController.java

```diff
@@ -139,4 +140,9 @@ public ResponseEntity<?> deleteProduct(@PathVariable Long productId) {
 
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
+
+    @PostMapping(value = "/purchase", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
It seems a bit weird to see Post without returning any object, but hey if the logic is here Im all for it! Just take care